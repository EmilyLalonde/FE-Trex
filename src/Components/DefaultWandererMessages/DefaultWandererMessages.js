import React, { Component } from "react";
import Footer from "../Footer/Footer";
import Header from "../Header/Header";
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity,
  TextInput,
  Keyboard
} from "react-native";

export default class DefaultWandererMessages extends Component {
  constructor() {
    super();
    this.state = {};
  }
  render() {
    return (
      <View>
      <View>
        <Header />
      </View>
      <View style={styles.container}>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 1</Text>
        </TouchableOpacity>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 2</Text>
        </TouchableOpacity>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 3</Text>
        </TouchableOpacity>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 4</Text>
        </TouchableOpacity>
      </View>
      <View>
        <Footer />
      </View>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#000000',
    alignItems: 'stretch',
    justifyContent: 'flex-start'
  },
  inputContainer: {
    marginTop: 15
  },
  textInput: {
    backgroundColor: "white",
    borderColor: "#CCCCCC",
    borderTopWidth: 1,
    borderBottomWidth: 1,
    height: 50,
    fontSize: 25,
    marginTop: 15,
    paddingLeft: 20,
    paddingRight: 20
  },
  button: {
    borderColor: "white",
    borderWidth: 1,
    borderRadius: 8,
    borderStyle: "solid",
    width: "auto",
    height: 60,
    margin: 20,
    fontSize: 30,
    padding: 10,
    color: "white",
    textAlign: "center",
    backgroundColor: "#1C4263"
  }
});
