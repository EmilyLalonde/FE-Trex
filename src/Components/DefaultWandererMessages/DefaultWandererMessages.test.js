import React from "react";
import "react-native";
import DefaultWandererMessages from "./DefaultWandererMessages";
import { shallow } from "enzyme";

describe("DefaultWandererMessages", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<DefaultWandererMessages navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
