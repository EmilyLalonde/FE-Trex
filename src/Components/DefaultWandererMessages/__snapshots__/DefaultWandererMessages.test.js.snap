// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultWandererMessages should match the snapshot when loading 1`] = `
<View>
  <View>
    <WandererHeader />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "backgroundColor": "#000000",
        "flex": 1,
        "justifyContent": "flex-start",
      }
    }
  >
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#1C4263",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 1
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#1C4263",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 2
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#1C4263",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 3
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#1C4263",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 4
      </Text>
    </TouchableOpacity>
  </View>
  <View>
    <WandererFooter />
  </View>
</View>
`;
